CMakeLists.txt

```diff
@@ -724,7 +724,7 @@ endif()
 ################################################################################
 # Documentation
 ################################################################################
-option(ENABLE_DOCS "Enable building documentation" ON)
+option(ENABLE_DOCS "Enable building documentation" OFF)
```

Comment:
This change is completely unnecessary. If Doxygen isn't found `ENABLE_DOXYGEN` gets forced to off. Also `the doc-doxygen` target is not part of the `all` target so won't get built by default anyway.